src/main/res/values-notnight/colors.xml

```diff
@@ -26,7 +26,7 @@
     <color name="accent">#328ce7</color>
     <color name="primary_text">#DE000000</color>
     <color name="secondary_text">#8A000000</color>
-    <color name="icons">#000000</color>
+    <color name="icons">#328ce7</color>
```

Comment:
Instead of hardcoding the values, you can use a reference to the accent colour so that it's easier to change the accent colour, if ever needed in future.

Reply:
@chirag-jn that is a good idea. I will do it.